core/conversion/converters/impl/expand.cpp

```diff
@@ -189,7 +188,7 @@ bool add_expand_dynamic(
   // min(1, sub(input_shape, 1))
   int32_t* one_vector_tmp = new int32_t[1];
   one_vector_tmp[0] = 1;
-  auto one_vector = vec2Tensor(one_vector_tmp, 1, ctx);
+  auto one_vector = util::arrToTensor(one_vector_tmp, 1, ctx);

```

Comment:
Can be replaced with tensor_to_const. Eg usage: [https://github.com/NVIDIA/TRTorch/blob/master/core/conversion/converters/impl/element_wise.cpp#L111](url)